README.md

```diff
@@ -82,51 +111,42 @@ public protocol Tool {
 
 Based on this, only `name` should be provided as well as `presentPreview:` method which will present the tool view controller from the `sourceScreenProvider` view controller defined in the Sentinel configuration.
 
-### Getting started
+## Contributing
 
-To add the *Sentinel* to your project, install the library via CocoaPods as instructed above. After that, define *Sentinel* configuration with mandatory parameters and call `Sentinel.shared.setup:` in the `AppDelegate` method `application(_:didFinishLaunchingWithOptions:)`.
+We believe that the community can help us improve and build better a product.
+Please refer to our [contributing guide](CONTRIBUTING.md) to learn about the types of contributions we accept and the process for submitting them.
 
-```swift
-let configuration = Sentinel.Configuration(
-    sourceScreenProvider: SourceScreenProviders.default,
-    trigger: Triggers.shake,
-    tools: [
-        GeneralInfoTool(),
-        UserDefaultsTool(),
-        LoggieTool(),
-    ],
-    preferences: [
-        OptionSwitchItem(
-            name: "Analytics",
-            setter: { AppSwitches.analyticsEnabled = $0 },
-            getter: { AppSwitches.analyticsEnabled },
-            userDefaults: .standard,
-            userDefaultsKey: "com.infinum.sentinel.optionSwitch.analytics"
-        )
-    ]
-)
+To ensure that our community remains respectful and professional, we defined a [code of conduct](CODE_OF_CONDUCT.md) that we expect all contributors to follow.
 
-Sentinel.shared.setup(with: configuration)
-```
-
-## Privacy
-
-Sentinel does not collect any user data. We have provided a [privacy manifest](https://github.com/infinum/ios-sentinel/blob/master/Sentinel/SupportingFiles/PrivacyInfo.xcprivacy) file that can be included in your app.
+We appreciate your interest and look forward to your contributions.
 
+## License
 
-## Authors
+```text
+Copyright 2024 Infinum
 
-* Vlaho Poluta, vlaho.poluta@infinum.com
-* Nikola Majcen, nikola.majcen@infinum.com
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
 
-## License
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
We should publish new version since we are changing license

Reply:
The plan is to put that to `2.0.0` release.
I will change the base branch (create a new release branch for that).